eventhubs/client/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/models/package-info.java

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+/**
+ * Package containing classes used for creating and configuring events that are being sent-to and received-from Azure Event Hubs Service.
```

Comment:
Our `.editorconfig` has a ruler at 120 characters. There are IntelliJ settings that can wrap comments exceeding that ruler.

Reply:
nit: Event Hubs Service -> Event Hubs service
To align with all the other usages of it.